NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/xcshareddata/xcschemes/iOS-Update plists.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1420"
```

Comment:
Looking at this, I feel like maybe a change I had done moved these versions forward. Are we going to be tug-of-war'ing these lines, or is there a better way and/or am I just misremembering?

Reply:
These are just because i am using the latest xcode. It's irrelevant really just means xcode wont give suggestions to upgrade the project

Reply:
sorry other way round. I guess the template project is not upgraded for xc 1420

Reply:
reverted those plist scheme changes